test/parsers.js

```diff
@@ -146,9 +147,10 @@ test('#getContextFromRequest()', function (t) {
   })
 
   t.test('full URI', function (t) {
+    var conf = { captureHeaders: true, captureBody: 'off' }
     var req = getMockReq()
     req.url = 'https://www.example.com:8080/some/path?key=value'
-    var parsed = parsers.getContextFromRequest(req)
+    var parsed = parsers.getContextFromRequest(req, conf)
```

Comment:
Since we're not actually testing the body or the headers, I'd just pass in an empty object here. Same goes for the other tests below where we don't need these. What do you think?

Reply:
fixed.